src/utils/resizeShadowCameraFrustum.js

```diff
@@ -53,7 +53,7 @@ export default function resizeShadowCameraFrustum(light, scene) {
   verts[6].set(max.x, max.y, min.z);
```

Comment:
maybe not a huge deal now that we are only doing this on scene load, but we compute the scene bounding box again for each light. Might be better to calculate that once and pass it into `resizeShadowCameraFrustum`